sample/build.gradle

```diff
@@ -48,6 +48,7 @@ dependencies {
     implementation "com.squareup.moshi:moshi-kotlin:$moshiVersion"
     kapt "com.squareup.moshi:moshi-kotlin-codegen:$moshiVersion"
     implementation "org.koin:koin-android-architecture:$koinVersion"
+    implementation "org.koin:koin-android:$koinVersion"
```

Comment:
I wonder  if the dependency for Koin `ext.koinVersion = '0.9.3'` should be updated to the latest `1.0.2`.